tools/js_manipulation.py

```diff
@@ -108,7 +108,10 @@ def make_dynCall(sig, args):
     else:
       return 'Module["dynCall_%s"](%s)' % (sig, args)
   else:
-    return 'getWasmTableEntry(%s)(%s)' % (args[0], ','.join(args[1:]))
+    arg0 = args[0]
+    if settings.MEMORY64:
+      arg0 = f'Number({arg0})'
+    return 'getWasmTableEntry(%s)(%s)' % (arg0, ','.join(args[1:]))
```

Comment:
if we're going to switch to f-strings in this function, might as well convert this line too?